lib/compilers/ldc.js

```diff
@@ -49,6 +50,40 @@ class LDCCompiler extends BaseCompiler {
     isCfgCompiler() {
         return true;
     }
+
+    couldSupportASTDump(version) {
+        const versionRegex = /\((\d\.\d+)\.\d+\)/;
```

Comment:
I trust you know the matching regex for LDC, but could you add a minimal test to `tests/d-tests.js` that more or less checks expected results for some versions? That would make it so we don't change it by accident

Reply:
I was wondering how to add tests for this (not just version test, but other testing too). I'lll add some simple tests in `d-tests.js`.